server.go

```diff
@@ -540,6 +540,20 @@ func (s *Server) errorf(format string, a ...interface{}) {
 	}
 }
 
+// ServiceRegistrar wraps a single method that supports service registration. It
+// enables users to pass concrete types other than grpc.Server to the service
+// registration methods exported by the IDL generated code.
+//
+// This API is EXPERIMENTAL.
+type ServiceRegistrar interface {
+	// RegisterService registers a service and its implementation to the
+	// concrete type implementing this interface. It is called from the IDL
+	// generated code.
+	//
+	// This API is EXPERIMENTAL.
+	RegisterService(*ServiceDesc, interface{})
```

Comment:
Name these parameters:

`desc *ServiceDesc, impl interface{}`

And document them above:

```go
// desc describes the service and its methods and handlers.  impl is the service
// implementation which is passed to the method handers*.
```

\* - for some reason we can't explain.

Reply:
Done.